components/db/DbProvider.jsx

```diff
@@ -11,14 +11,14 @@ import DbContext from '@db/DbContext';
 import moment from 'moment';
 
 // List of remote dataset locations used by db.registerFileURL
-const datasets = {
+const prod_datasets = {
```

Comment:
Please make this one unified object, and use string interpolation for the HF repo name (e.g. `311-data` vs `311-Data-Dev`